keyboards/4pplet/waffling60/rev_e/info.json

```
@@ -0,0 +1,649 @@
+{
+    "manufacturer": "4pplet",
+    "keyboard_name": "waffling60 Rev E",
+    "maintainer": "4pplet",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "A2", "pin_b": "A1", "resolution": 2}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
```

migrated from `rules.mk`
```suggestion
        "extrakey": true,
        "key_lock": true,
```